api.visual/src/org/netbeans/api/visual/widget/Widget.java

```diff
@@ -214,6 +215,7 @@ public final void addChild (Widget child) {
      */
     public final void addChild (Widget child, Object constraint) {
         assert child.parentWidget == null;
+        assert SwingUtilities.isEventDispatchThread();
```

Comment:
Isn't `EventQueue.isEventThread()` the preferred way since its introduction? We might have a hint to suggest a rewrite. what do you think about such idea, @jlahoda?

Reply:
Correct, but SwingUtilities are used in other places for e.g. invokeLater/AndWait - is there a consensus in NB's codebase on that ? If so, I should probably change the other places, too.